src/vm/object.cpp

```diff
@@ -1658,9 +1658,9 @@ void __fastcall ZeroMemoryInGCHeap(void* mem, size_t size)
         *memBytes++ = 0;
 
```

Comment:
This method is only used to clear structs in a few places in reflection now. Writing assembly code to do that is not worth it.